src/components/GridListCustom.js

```diff
@@ -6,21 +6,25 @@ import styles from '../styles';
 
 function GridListCustom(Component) {
   return props => {
-    return (
-      <div style={styles.gridListContainer}>
-        <GridList cellHeight={180} style={styles.gridList}>
-          <Subheader>{props.header}</Subheader>
-          {Object.keys(props.data)
-            .reverse()
-            .map((key, index) => {
-              let tile = props.data[key];
-              return (
-                <Component {...props} image={tile} key={key} keyProp={key} />
-              );
-            })}
-        </GridList>
-      </div>
-    );
+    if (props.data !== null) {
+      return (
+        <div style={styles.gridListContainer}>
+          <GridList cellHeight={180} style={styles.gridList}>
+            <Subheader>{props.header}</Subheader>
+            {Object.keys(props.data)
+              .reverse()
+              .map((key, index) => {
+                let tile = props.data[key];
+                return (
+                  <Component {...props} image={tile} key={key} keyProp={key} />
+                );
+              })}
+          </GridList>
+        </div>
+      );
+    } else {
+      return <div style={styles.gridListContainer}> </div>;
```

Comment:
Put a message here that you don't have anything uploaded.

Reply:
Emm, you bad bad.
lol